packaging/datadog-agent/source/supervisord.conf

```diff
@@ -45,7 +45,7 @@ command=python agent/jmxfetch.py
 stdout_logfile=supervisord/logs/jmxfetch.log
 redirect_stderr=true
 priority=999
-startsecs=0
+startsecs=3
```

Comment:
I think what you really want to implement is "startretries".

Also you should likely implement "exitcodes" and exit with a given exit code that would be on that list when java cannot be found.